### PR TITLE
Don't rely on outputs order from adapter in cls wrapper

### PR DESCRIPTION
**Pull request by sovrasov**
# What does this PR do?

OVMS adapter can return the names in an arbitrary order, while the cls wrapper expects logits to be at the first place.

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by sovrasov** [comment]
@RHeckerIntel please have a look

**Review by Wovchena — APPROVED** [review]
